src/common/transformations/tests/common_optimizations/align_mixed_fp32_fp16_types_test.cpp

```diff
@@ -0,0 +1,306 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <openvino/core/model.hpp>
+#include <openvino/opsets/opset8.hpp>
+#include <openvino/pass/manager.hpp>
+#include <transformations/common_optimizations/align_mixed_fp32_fp16_types.hpp>
+#include <transformations/common_optimizations/mark_subgraphs_to_keep_in_mixed_precision.hpp>
+#include <transformations/convert_precision.hpp>
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+
+using namespace testing;
+using namespace ov;
+using namespace std;
```

Comment:
please add `using namespace ov::opset10;` and do not mention `opset10` below during nodes creation

Reply:
done